examples/ckbtc/README.md

```diff
@@ -1,120 +1,79 @@
+# Chain Key BTC
+
 Keep in mind that this is a simple demo example of ckBTC with Azle, and is probably not suitable for production use. Learn wisely.
 
 This ckBTC example shows you how to setup ckBTC locally, with the `ckBTC ledger`, `internet identity`, `kyt`, `minter`, and `bitcoind`. It also has a canister wallet backend and frontend. The canister wallet shows how a canister can control a number of ckBTC subaccounts. The frontend functionality only shows how to mint ckBTC and transfer between other canister wallets.
 
-# Installation
+## Setup
 
-## bitcoind
+Install all npm dependencies and download canister WASMs binaries:
 
 ```bash
-mkdir .bitcoin
-mkdir .bitcoin/data
-
-curl https://bitcoincore.org/bin/bitcoin-core-23.0/bitcoin-23.0-x86_64-linux-gnu.tar.gz -o bitcoin.tar.gz
-
-tar xzf bitcoin.tar.gz --overwrite --strip-components=1 --directory=.bitcoin/ bitcoin-23.0/bin/
-
-rm -rf bitcoin.tar.gz
+npm install
 ```
 
-## ckbtc ledger canister
+Update the `canisters.wallet_backend.declarations.node_compatibility` property in dfx.json to `false`:
```

Comment:
Are all of these images necessary? It's a standard internet identity application, and it's very simple

Reply:
Because the frontend redirects you to the II application I thought it would be helpful. We can definitely remove if you'd like. We discussed in person and it sounds ok for now. If there are IP issues we can remove them later.